js/module.js

```diff
@@ -0,0 +1,156 @@
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/Tosin.jpg',
+    name: 'Tosin Adewale',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/mama.jpg',
+    name: 'Samuel Juliana',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/omos.jpg',
+    name: 'Omos Ehigbe',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/Tosin.jpg',
+    name: 'Iya Alata',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/mama.jpg',
+    name: 'Mama Omos',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+];
+
+const sections = {
+  program: document.querySelector('#main-program'),
+  join: document.querySelector('join'),
+  sponsor: document.querySelector('#sponsors'),
+  news: document.querySelector('#news'),
+};
+
+const checkbox = document.querySelector('#menu-checkbox');
+const body = document.querySelector('body');
+
+function createSpeakerElement(speaker) {
+  const article = document.createElement('article');
+  article.classList = 'speaker';
+
+  const img = document.createElement('img');
+  img.className = 'spk-img';
+  img.src = speaker.imgSrc;
+  img.alt = speaker.name;
+
+  const section = document.createElement('section');
+  section.className = 'spk-bio';
+
+  const h2 = document.createElement('h2');
+  h2.className = 'spk-name';
+  h2.textContent = speaker.name;
+
+  const pQualities = document.createElement('p');
+  pQualities.className = 'spk-qualities';
+  pQualities.textContent = speaker.qualities;
+
+  const pSpeech = document.createElement('p');
+  pSpeech.className = 'spk-speach';
+  pSpeech.textContent = speaker.speech;
```

Comment:
- [ ] 👍 Great job on dynamically generating the speakers! However, the color for this paragraph does not match the design. Please update your code to use the correct color as per the design.

By ensuring that the paragraph  color aligns with the design, your project will look more visually appealing and consistent. Keep up the good work! 🎨🌟🚀